jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -416,6 +416,41 @@
 					"response": []
 				},
 				{
+					"name": "create group2 for duplicate error check",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"var jsonData = JSON.parse(responseBody);",
+									"postman.setEnvironmentVariable(\"GroupId2\", jsonData.applicationResponseContent.id.id);",
+									"tests[\"Successful POST request\"] = responseCode.code === 201;",
```

Comment:
This test is confusing: the name and description are called "duplicate error check" but here the response is expected to be 201 and the ID returned is being saved off as an environment variable.

So is this test expected to successfully create the group or throw an error?